### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by athar05**
This branch contains following features: 

Create a post
Like a post
Comment on post
Edit & Delete post
User feed
Sort by Date and Trending
Follow/Unfollow
Bookmark Post
Authentication
Sign-up
Login
Logout

Do not check: 

Mobile responsiveness

### CODE REVIEW

File: src/backend/controllers/PostController.js [hunk 1]
```diff
@@ -180,10 +181,13 @@ export const likePostHandler = function (schema, request) {
       (currUser) => currUser._id !== user._id
     );
     post.likes.likeCount += 1;
-    post.likes.likedBy.push(user);
+    post.likes.likedBy.push({...user, bookmarks: []});
+    // post.likes.likedBy.push({...rest});
+    console.log(rest)
```
Comment (Sen-442b) [inline src/backend/controllers/PostController.js:186]: You can remove redundant debuggers

File: src/components/Navbar.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { AppBar, Toolbar, IconButton, Typography, Stack, Button } from '@mui/mat
 import ConnectWithoutContactIcon from '@mui/icons-material/ConnectWithoutContact';
 import {Link, useNavigate} from 'react-router-dom'
 import { useDispatch } from 'react-redux';
-import { signOut } from '../features/auth/authSlice';
+import { signOut } from '../features/slices/authSlice';
 
 const Navbar = () => {
```
Comment (Sen-442b) [inline src/components/Navbar.js:9]: You have an empty component here

File: src/components/posts/posts.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+.posts:hover {
+    cursor: pointer;
+    background-color: #e9efef;
+}
+
+.posts-body {
+    flex: 1;
+    padding: 0.5rem 0.5rem;
+    width: min-content;
+    /* border: 1px solid black; */
```
Comment (Sen-442b) [inline src/components/posts/posts.css:18]: you can remove unused stylings

**Review by Sen-442b — APPROVED** [review]
- Code Looks Clean
- Proper Folder Structure

File: src/backend/controllers/PostController.js (hunk 1 above)
Comment (athar05) [reply src/backend/controllers/PostController.js:186]: Done

**Pull request merged**